src/maker/rpc/messages.rs

```diff
@@ -36,5 +39,27 @@ pub enum RpcMsgResp {
     NewAddressResp(String),
     SendToAddressResp(String),
     GetTorAddressResp(String),
-    GetDataDirResp(String),
+    GetDataDirResp(PathBuf),
+    Shutdown,
+}
+
+impl Display for RpcMsgResp {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Pong => write!(f, "Pong"),
+            Self::NewAddressResp(addr) => write!(f, "{}", addr),
+            Self::SeedBalanceResp(bal) => write!(f, "{} sats", bal),
```

Comment:
The reason for this change is that `Amount` in debug mode -> shows in `SAT` and `BTC` for display.
- This is working here because that `bal` is in `u64` not `Amount`.
- While in Taker cli app -> The amount will show like `x SAT`
https://github.com/citadel-tech/coinswap/blob/b80244a5f4bc8c5ee604ac5a2818710bfe1ed379/src/bin/taker.rs#L186